pkg/rancher-desktop/backend/wsl.ts

```diff
@@ -1197,10 +1195,19 @@ export default class WSLBackend extends events.EventEmitter implements VMBackend
           DISTRO_DATA_DIRS: DISTRO_DATA_DIRS.join(':'),
           LOG_DIR:          paths.logs,
         },
-        stdio:       ['ignore', stream, stream],
+        stdio:       ['ignore', 'pipe', 'pipe'],
         windowsHide: true,
       });
+    for (const readable of [this.process.stdout, this.process.stderr]) {
+      if (readable) {
```

Comment:
Why do we need this check? How can either of these streams be undefined?

Reply:
It can't; it exists to satisfy TypeScript because _that_ isn't aware that `.stdout` and `.stderr` are always non-null in this case.